warnings.cpp

```diff
@@ -0,0 +1,21 @@
+// warnings.cpp
+#include "./warnings.h"
+#include <iostream>
+#include <thread>
+#include <chrono>
+
+using std::cout;
+using std::flush;
+using std::this_thread::sleep_for;
+using std::chrono::seconds;
+
+// Function to display a blinking warning message
+void displayWarning(const std::string& message) {

```

Comment:
Good approach to separate this into its own file. Then the mechanism of message-display can evolve without disturbing the rest of the functionality.